test/fixtures/calendar_title_types.yml

```diff
@@ -0,0 +1,114 @@
+one:
+  type_id: 1
+  title_id: 3
+  status: 0
+
+two:
+  type_id: 1
+  title_id: 4
+  status: 0
+
+three:
+  type_id: 1
+  title_id: 5
+  status: 0
+
+four:
+  type_id: 1
+  title_id: 6
+  status: 0
+
+five:
+  type_id: 1
+  title_id: 7
+  status: 0
+
+six:
+  type_id: 1
+  title_id: 8
+  status: 0
+
+seven:
+  type_id: 1
+  title_id: 9
+  status: 0
+
+eight:
+  type_id: 1
+  title_id: 10
+  status: 0
+
+nine:
+  type_id: 1
+  title_id: 11
+  status: 0
+
+ten:
+  type_id: 1
+  title_id: 12
+  status: 0
+
+eleven:
+  type_id: 1
+  title_id: 13
+  status: 0
+
+twelve:
+  type_id: 1
+  title_id: 14
+  status: 0
+
+thirteen:
+  type_id: 1
+  title_id: 15
+  status: 0
```

Comment:
Buralarda enumların isimlerini kullansan daha güzel olur. Örneğin ben active'i 1, passive'i 0 yapalım dedim, şimdi gelip tüm bu kodu değiştirmek zorunda kalacaksın. Oysa ki şöyle olsa gerek olmayacaktı:

`status: active`

Reply:
Çok mantıklı :+1: